### DISCUSSION TITLE
Moderator Features

### DISCUSSION THREAD

**Discussion by kaiser1245** [aeharding/voyager#88] [Ideas]
Is there currently any plan to add mod actions to posts for communities you moderate? Also a way to view reports would be super helpful.

**Comment by aeharding** [comment]
Definitely. (Not until other priorities are done though.) I've spent my fair share in the apollo/old reddit mod queue, but I have no mod experience with lemmy.

If anyone is willing to adopt me as a mod to a larger lemmy community for a few weeks so I can get an idea of the differences between lemmy and reddit moderation, that would help hugely.

**Comment by AeonFX** [comment]
Hello Alex,
I am a mod of one of the bigger  communities and would be willing to let you be mod there for some time. However it probably would just be easier to create a test community and create some reports there to see how Voyager/Lemmy handles this. There seemed to be some issues with federation of mode reports though, some seem to be fixed but apparently resolving a report from a remote instance still might be problematic: https://lemmy.world/post/20733887

Anyway the biggest problem imo is that Voyager doesn't have any system notifications implemented yet (at least not on Android). Obviously this also would make more sense if lemmy had actual push notification implemented. At least when I actively opened the app and just pulled all changes it should give me options for system notifications. Or even better have some polling functionality like the boost app hast where you can set how often the app polls the backend, with a minimum of 30min to not put too much strain on the server.

So in case of a new message/report in my lemmy inbox voyager does not show me a notification on my phone. Also it seems that what you called "moderation zone" where I can see the "mod queue" with open reports also has not in-app indicators. While I do see an indicator for new message/replies etc. it seems that the mod zone is treated differently and I have to always actively click on "mod zone" -> "mod queue". 
Also the "mod logs" section does not allow to show me only the mod log of communities I actually moderate but rather shows instance wide logs (actually it seems to be a list of all federated mod message, even remote instances). 
Obviously for mods it would be good to see the log of only instances I mod or at least give me an option to filter it out.
